src/App.tsx

```diff
@@ -1,12 +1,62 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import { makeStyles } from "@mui/styles";
+import { Button } from "@mui/material";
+import { Box } from "@mui/system";
 
 
-function App() {
+const useStyles = makeStyles({
+  container: {
+    width: '90%',
+    margin: '50px auto',
+    textAlign: 'center'
+  },
+  h1: {
+    color: 'burlywood',
+    fontSize: '3rem',
+  },
+  board: {
+    maxWidth: '450px',
+    margin: '25px auto',
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  square: {
+    width: '150px',
+    height: '150px',
+    border: '2px solid burlywood',
+    cursor: 'pointer'
+  },
+});
+
+const App = () => {
+  const [boardTable, setBoardTable] = useState(Array(9).fill(null));
+  const [turn, setTurn] = useState(false);
+
+  const classes = useStyles();
+
   return (
-    <h1>Tic Tac Toe with Typescript</h1>
+    <div className={classes.container}>
+      <h1 className={classes.h1}>Tic Tac Toe with Typescript</h1>
+      <Box mt={2}>
+        <Button
+          variant="outlined"
+          size="medium">
+          Reset
+        </Button>
+      </Box>
+      <div className={classes.board}>
+        {
+          boardTable.map((board, index) => (
+            <div
+              className={classes.square}
+              key={index}>
+              {board}
+            </div>
+          ))
```

Comment:
Board table is redundant. Either call it board or table, you can also call it grid. Since map gets one element at a time, each of these elements should be treated as what they are, in this case a cell, this way it's clearer that you're iterating over cells and passing the cell info to a child component.

Reply:
Ok ok, I renamed the state to "board" and the map parameter to cell